### ISSUE TITLE
Performance regression in version 1.5.1

### ISSUE THREAD

**Issue by mjkaye** [TheGammaSqueeze/GammaOS#169]
In the video, you can see the performance difference between two RG405M devices running GammaOS Lite versions 1.3.3 and 1.5.1, respectively. The video shows the same scene on each, from Twilight Princess running on Dolphin.

https://github.com/TheGammaSqueeze/GammaOS/assets/1014467/ed785e6b-fcce-4510-a19f-b8418a540153

Both devices have a clean install of their respective GammaOS version. Both are running Dolphin 5.0-21395 with the same settings. Both devices are set to "Max Performance".

Please let me know if there's any more information I can provide that could help with diagnosing the cause.

**Comment by mjkaye** [comment]
The same performance regression can be observed with PS2. In this video, I'm using NetherSX2 on both devices. As before, the only difference is that the grey device is running GammaOS 1.3.3. whereas the black device is running 1.5.1. The NetherSX2 settings on both are as follows:

- Settings -> Graphics -> GPU Renderer: Vulkan
- Settings -> System Settings -> EE Cycle Rate: -2
- Settings -> System Settings -> EE Cycle Skip: 2
- Settings -> Graphics Settings -> Hardware Download Mode: Disable readbacks


https://github.com/TheGammaSqueeze/GammaOS/assets/1014467/79a0dd22-903a-4f72-9818-ae9697fa6997

Is there anything else I can do to help with this?

**Comment by mjkaye** [comment]
@TheGammaSqueeze Thank you for looking into this.

I'm happy to give those a test, but I'm not very familiar with Magisk. I want to make sure I do the tests properly, so I'd be grateful if you could give me some pointers about how to install those modules.

Thanks.

**Comment by mjkaye** [comment]
It turns out my testing was not as thorough as I had thought.

Although I had tested 1.3.3 with multiple devices, it happened that the device I had chosen for my version 1.5.1 testing had performance issues. I have now tested that device with version 1.3.3 and it had the same performance issues as with 1.5.1. I have now tested two other devices with both version 1.3.3 and 1.5.1 and found no performance regression.

I apologize for the incorrect issue report.
